src/libcore/borrow.rs

```diff
@@ -129,6 +129,7 @@ impl<T> ToOwned<T> for T where T: Clone {
 ///     }
 /// }
 /// ```
+//#[deriving(Show)] NOTE(stage0): uncomment after snapshot
```

Comment:
I meant after a snapshot of this commit. Currently, the `fmt::Show` of below is depended on for something (I don't recall), but after stage0, this can simply be a derived implementation afterward, showing if its `Borrowed` or `Owned`.